doc/topics/releases/3006.rst

```diff
@@ -0,0 +1,137 @@
+.. _release-3006:
+
+=========================================
+Salt 3006 release notes - Codename Sulfur
+=========================================
+
+Salt 3006 is currently under development.
+
+
+Non-Tiamat packaging support
+----------------------------
+The non-Tiamat packaging system previously used for Salt will only be provided
+for platforms that support Python 3.7 and above. If a platform is using
+Python 3.5 or 3.6, only Tiamat packages will be available for that platform.
+This policy change will require users to update their repo files to point to
+the new repo path to use the new Tiamat packages. The full installation instructions
+can be found at https://repo.saltproject.io. This change impacts various OS platforms, including:
```

Comment:
**Suggestion (blocking):** The install directions are going to be deprecated from repo. We should link to the install guide instead.